docs/index.rst

```diff
@@ -14,57 +15,40 @@ Django GUID
-
-
-Example
--------
+Django GUID attaches a unique correlation ID to all your log outputs for every requests you handle. This means that every error now has an ID connecting it to all other relevant logs, making
+debugging simple.
 
-Using ``demoproj`` as an example, all the log messages **without** ``django-guid`` would look like this:
+Integration are also made possible, as the ID can be *returned as a header*,
+or can be added as an *outgoing header* in external requests, making it possible to extend the reach of correlation IDs to whole systems.
 
-.. code-block:: bash
+**Resources**:
 
-    INFO 2020-01-14 12:28:42,194 django_guid.middleware No Correlation-ID found in the header. Added Correlation-ID: 97c304252fd14b25b72d6aee31565843
-    INFO 2020-01-14 12:28:42,353 demoproj.views This is a DRF view log, and should have a GUID.
-    INFO 2020-01-14 12:28:42,354 demoproj.services.useless_file Some warning in a function
-
-With ``django-guid`` every log message has a GUID (``97c304252fd14b25b72d6aee31565843``) attached to it,
-through the entire stack:
-
-.. code-block:: bash
+* Free software: BSD License
+* Documentation: https://django-guid.readthedocs.io
+* Homepage: https://github.com/JonasKs/django-guid
 
-    INFO 2020-01-14 12:28:42,194 [None] django_guid.middleware No Correlation-ID found in the header. Added Correlation-ID: 97c304252fd14b25b72d6aee31565843
-    INFO 2020-01-14 12:28:42,353 [97c304252fd14b25b72d6aee31565843] demoproj.views This is a DRF view log, and should have a GUID.
-    INFO 2020-01-14 12:28:42,354 [97c304252fd14b25b72d6aee31565843] demoproj.services.useless_file Some warning in a function
+**Examples**
 
-For multiple requests at the same time over multiple threads, see the :ref:`extended_example`.
+Log output with a GUID:
 
-Why
----
+.. code-block::
 
-``django-guid`` makes it extremely easy to track exactly what happened in any request. If you see one error
-in your log, you can use the attached GUID to search for any connected log message to that single request.
-The GUID can also be returned as a header and displayed to the end user of your application, allowing them
-to report an issue with a connected ID. ``django-guid`` makes troubleshooting easy.
+    INFO ... [773fa6885e03493498077a273d1b7f2d] project.views This is a DRF view log, and should have a GUID.
+    WARNING ... [773fa6885e03493498077a273d1b7f2d] project.services.file Some warning in a function
+    INFO ... [0d1c3919e46e4cd2b2f4ac9a187a8ea1] project.views This is a DRF view log, and should have a GUID.
+    INFO ... [99d44111e9174c5a9494275aa7f28858] project.views This is a DRF view log, and should have a GUID.
+    WARNING ... [0d1c3919e46e4cd2b2f4ac9a187a8ea1] project.services.file Some warning in a function
+    WARNING ... [99d44111e9174c5a9494275aa7f28858] project.services.file Some warning in a function
 
 
-Contents
---------
+Log output without a GUID:
 
-.. toctree::
-    :maxdepth: 3
+.. code-block::
 
-    install
```

Comment:
I'd like the TOC there tbh. It might not give a lot of visible value, but there's many `CTRL`+`F`ers out there, including me.

Reply:
I agree - missed that and just copy pasted over it. 

Added it back in